### ISSUE TITLE
Firefly wallet backup issue

### ISSUE THREAD

**Issue by iotaislove** [iotaledger/firefly#719]
## Issue description:  Whenever I want to save or skip my stronghold backup (before using the wallet), there is an issue saying that an error has occured. And so I cannot register for the wallet

Briefly describe the issue.

## Wallet version

Which version of wallet are you using? FIREFLY

- wallet version (version number, commit, or branch):

## System specification

What is your operating system?

Mac/Win/Linux (+Version and or Distro if possible)  WINDOWS 10

## Is it a repeatable issue?

Y/N YES
## Steps To reproduce the issue

Explain how the maintainer can reproduce the issue.

1.
2.
3.

## Expected behaviour

Describe what you expect to happen.

## Actual behaviour

Describe what actually happens.

## Errors: Cannot save or skip the stronghold backup (a red warning in the right corner)

Paste any errors that you see from the Error Log in the Firefly menu or share screenshots of the issue.

**Comment by here-in-now** [comment]
Hi, me is the same.
Since yesterday I try to make an account to firefly but always at the end, after taking the Pass Phrase and trying to backup with Stronghold or also without doing this, ther I get the same issue and I get no account. Starting from new, always the same ending.
I can´t use the password and acccountname I gave before, there is always to start from new.
System: Win10 newest, Wallet: 0.3.1
Please help in easy words if you can ;-)))

**Comment by Phyloiota** [comment]
please try this:
1.) create an account without the use of any special characters in the account name (like <>#@%*) just letters and numbers.
2.) Uninstall firefly, dleter the firefly folder in ([your username]/appdata/roaming/firefly) and run the firefly installation (.exe file) as administrator (right click on the .exe file => run as administrator.

Report if its still not working please

**Comment by iotaislove** [comment]
I unistalled and installed it again today and it works fine :)

**Comment by iotaislove** [comment]
> Hi, me is the same.
> Since yesterday I try to make an account to firefly but always at the end, after taking the Pass Phrase and trying to backup with Stronghold or also without doing this, ther I get the same issue and I get no account. Starting from new, always the same ending.
> I can´t use the password and acccountname I gave before, there is always to start from new.
> System: Win10 newest, Wallet: 0.3.1
> Please help in easy words if you can ;-)))

uninstall and install it again, it will work fine :)

**Comment by cekrdoso** [comment]
Hi there,
Just installed (not actually, just run the AppImage file) Firefly (version 0.3.1) wallet for the fist time in my computer, OS Ubuntu 20.10. In the first attempt I've got the error "Something went wrong" and, at the console, "StrongholdError: snapshot password not set". Btw, no special characters in my account name. Then I closed the App and started again without doing anything else. Most of the options were already setted (like dark theme). But it was successful this time. So, I guess there still something going on.
